dspace-server-webapp/src/test/java/org/dspace/app/rest/EPersonRestRepositoryIT.java

```diff
@@ -1410,14 +1411,14 @@ public void patchCertificateNonAdminUser() throws Exception {
     }
 
     @Test
-    public void patchPasswordMissingValue() throws Exception {
```

Comment:
the scenario in the original test is loss. What happen if we try to set an empty password?

Reply:
I agree with @abollini that we should make sure to add a test here to prove what happens if you send an `ADD` with an empty password (I'd expect it to throw an error like before, but would be good to prove).

(Sidenote: @Raf-atmire  I've already done a quick code review & this is looking great to me, other then this one missing test.  I'll be waiting to submit my review though until I test it alongside the frontend PR)